up-l2/rpcclient.adoc

```diff
@@ -41,21 +41,38 @@ SPDX-License-Identifier: Apache-2.0
 
 All link:../languages.adoc[language-specific uProtocol libraries] *MUST* declare the `RpcClient` interface with method:
 
-`OUT Future link:../basics/umessage.adoc[UMessage] invokeMethod(IN link:../basics/uri.adoc[UUri], IN UPayload?, IN link:../basics/uattributes.adoc[CallOptions])`
+[source]
+----
+invokeMethod(
+  IN method: UUri,
+  IN options: CallOptions,
+  IN payload: UPayload?,
+  OUT result: Future<UMessage, UStatus>
+)
+----
 
 .Parameters
-[#parameters, width="80%",cols="30%,70%"]
+[#parameters, width="100%",cols="15%,25%,60%"]
 |===
 
-| Parameter
-| Description
+| Parameter | Type | Description
+
+| method
+| link:../basics/uri.adoc[UUri]
+| The URI representing the service method to invoke.
 
-| `UUri`
-| Method URI that is being invoked
+| options
+| CallOptions
+| Additional meta data to include in the request message.
 
-| `CallOptions`
-| Set of attributes that can be sent as part of a method invocation
+| payload
+| link:../basics/upayload.adoc[UPayload]
```

Comment:
@sophokles73 can you update this to reflect the UPayload change merged in just now? either we define what UPayload is or we pass payload and UPayloadFormat as 2 separate parameters

Reply:
This will actually need more changes because the CallOptions are gone as well, aren't they? FMPOV this should be done in another PR that deals with the l2 APIs ...